src/Microsoft.IdentityModel.Protocols/Configuration/ConfigurationManager.cs

```diff
@@ -152,7 +152,7 @@ public TimeSpan RefreshInterval
         /// <remarks>If the time since the last call is less than <see cref="AutomaticRefreshInterval"/> then <see cref="IConfigurationRetriever{T}.GetConfigurationAsync"/> is not called and the current Configuration is returned.</remarks>
         public async Task<T> GetConfigurationAsync()
         {
-            return await GetConfigurationAsync(CancellationToken.None);
+            return await GetConfigurationAsync(CancellationToken.None).ConfigureAwait(false);
```

Comment:
nit: you can remove `async` and directly return the task here, without having to use `ConfigureAwait(false);`.